### ISSUE TITLE
I managed to install it, but what after ?

### ISSUE THREAD

**Issue by konfiot** [devthane/docker-flamenco#5]
Hello !
First of all : Awesome work, I've struggled for a long time to setup flamenco server without finding anything appropriate, and what you've got there is amazing !
So I've setup the container, everything is working fine, but now I want to setup the manager, and when I try to link it, I'm redirected to <Server>/login which doesn't redirect to /login/local, and If I do it manually, I'm never redirected back to the setup of the manager =/

Also, when I try to launch the manager after setting just the URL (so without any secret), I get a 404 when it requests the URL  (but it works if I try to POST on URL http://<server>/api/flamenco/managers/task-update-batch (without two slashes)) =/

Again, thanks for this work, that's just awesome !

**Comment by devthane** [comment]
ok, I'm not 100% on your second problem, but I'm pretty sure the first issue has to do with the authentication. Solving it might resolve the other problem as well, or at least bring me close to figuring it out.

I created a task over at blender's repository at https://developer.blender.org/T54587 if you wanna follow it there.

I think that's the biggest hurdle to getting this to work.

**Comment by sr9yar** [comment]
Hello, do I understand it correctly, that due to auth issues it is impossible to use this repo locally ?

**Comment by devthane** [comment]
@sr9yar yes, I'm sorry without some collaboration from the flamenco team, I haven't been able to get this working, short of forking their project.

If you want to nudge this along you could comment on that issue I linked [https://developer.blender.org/T54587](https://developer.blender.org/T54587) on their end. Once that is resolved I believe I can get this working.

**Comment by devthane** [comment]
I think they are probably really busy with Blender 2.8

**Comment by devthane** [comment]
I’ve updated the reader to be more clear.

**Comment by sr9yar** [comment]
@devthane Thanks a lot for your reply. Yes, I've read this thread. I'm looking for a way to set up a small cloud render farm. I think I could look into this issue, but I'm not familiar pillars, blender id and flamenco itself and I can't spend a lot of time on it. Otherwise I could write docker containers myself. Right now all this seems to be an overkill, becuase I need only very basic funtionality. Having a ready containers set up like yours would be great, because it would kick start people like me into using flamenco.

Ps
In fact, the containers themselves run ok, but I can't get past login.

**Comment by devthane** [comment]
Yeah you are running into the same problems I did.

I spent a lot of time getting this project this far when i ran into that issue. I didn’t anticipate it taking this long for them to respond, but I’ve been busy with other things as well.

I personally only need a render farm for a few months a year. We are approaching that time again.

We could perhaps go ahead and implement the changes that are needed in pillar / flamenco and then worry about getting them merged upstream later.

I’ll do my best to look into this further, but I have a full time job and also am developing a site for another customer right now as well on the side, so bare with me. (Also python isn't my language of choice and I've not worked with django except for on this project.)

I’ll try to look at it this weekend and get a better idea of what’s needed at least and lay it out better.
